test/e2e/backup-restore/backup_restore_test.go

```diff
@@ -80,6 +82,9 @@ func testBackupRestore(t *testing.T, clusterSpec clusterSpec, clusterSize, numEn
 	tutils.ExpectNoError(testKube.Kubernetes.Client.Get(context.TODO(), types.NamespacedName{Namespace: namespace, Name: sourceCluster}, infinispan))
 
 	// Validate the number of entries stored in the someCache backup file
+	// Utilise a 15.0.x image as this still contains the unzip package
+	imageOp, err := tutils.VersionManager().LatestUpstreamPatch(semver.Version{Major: 15, Minor: 0})
```

Comment:
This looks like a temporary solution only. Once we remove 15.0.x from the available image, it'll fail. Even though this situation is a long time ahead of us.

Reply:
Yeah, my thoughts that's like years away and we can worry about that at the time. I didn't want to pull another image in, e.g. ubi, as that would then have to be added to the image cache.